sympy/sets/tests/test_sets.py

```diff
@@ -941,3 +942,13 @@ def test_issue_10113():
 def test_issue_10248():
     assert list(Intersection(S.Reals, FiniteSet(x))) == [
         And(x < oo, x > -oo)]
+
+
+def test_set_in_interval():
+    x = Symbol('x', real=True)
+    assert x in Interval(x, x+4)
+    assert Interval(1, 2) not in Interval(0, 5)
```

Comment:
Okay.
I thought that by `in`, maybe we should check if a set is a subset of the other.
But now, I get the point across and I think we should check for FiniteSet of integers in the Interval too.